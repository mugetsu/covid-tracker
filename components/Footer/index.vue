<template>
  <div class="wrap-footer">
    <footer>
    </footer>
  </div>
</template>

<script>
export default {
  name: 'Footer',
  data() {
    return {}
  }
}
</script>

<style lang="scss" scoped>
.wrap-footer {
  position: absolute;
  z-index: 1;
  padding: 0 16px;
  flex-shrink: 0;

  footer {
    font-weight: 300;
    font-size: 1rem;
    text-align: right;
    text-transform: uppercase;
    color: #ffffff;
  }

  p {
    // margin-bottom: 30px;
  }

  a {
    display: inline-block;
    font-weight: 400;
    text-transform: uppercase;
    text-decoration: none;
    color: #ffffff;
  }
}
</style>