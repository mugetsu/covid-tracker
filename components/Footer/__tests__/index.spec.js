import { shallowMount } from '@vue/test-utils'
import Footer from '../index.vue'

const factory = () => {
  return shallowMount(Footer, {})
}

describe('Footer', () => {
  test('mounts properly', () => {
    const wrapper = factory()
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  test('renders properly', () => {
    const wrapper = factory()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
