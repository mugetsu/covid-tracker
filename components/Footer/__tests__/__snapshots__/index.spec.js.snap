// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders properly 1`] = `
<div class="wrap-footer">
  <footer>
    <search-stub></search-stub>
  </footer>
</div>
`;
